noxfile.py

```diff
@@ -0,0 +1,74 @@
+import nox
+from nox_poetry import Session, session
+
+nox.options.reuse_existing_virtualenvs = True
+nox.options.sessions = ["fmt_check", "type_check", "lint", "test", "docs"]
+
+
+@session(python=["3.8", "3.9", "3.10"])
+def test(s: Session) -> None:
+    s.install(".", "pytest", "pytest-cov")
+    s.run(
+        "python",
+        "-m",
+        "pytest",
+        "--cov",
+        "fact",
+        "--cov-report",
+        "html",
+        "--cov-report",
+        "term",
+        *s.posargs,
+    )
+
+
+# For some sessions, set venv_backend="none" to simply execute scripts using the existing
+# environment. This requires that nox is installed and run using `poetry install`.
```

Comment:
Is this suggestion correct?

```suggestion
# environment. This requires that nox is installed and run after `poetry install`.
```

Reply:
Not quite, but it's not super clear so I will try to rephrase.